applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.hpp

```diff
@@ -11,54 +11,43 @@
 //                   Vahid Galavi
 //
 
-#if !defined(KRATOS_GEO_SMALL_STRAIN_U_PW_DIFF_ORDER_ELEMENT_H_INCLUDED)
-#define KRATOS_GEO_SMALL_STRAIN_U_PW_DIFF_ORDER_ELEMENT_H_INCLUDED
-
-#include "custom_retention/retention_law.h"
-#include "geometries/geometry_data.h"
-#include "includes/constitutive_law.h"
-#include "includes/define.h"
-#include "includes/element.h"
-#include "includes/kratos_export_api.h"
-#include "includes/serializer.h"
-#include "includes/smart_pointers.h"
-#include "includes/ublas_interface.h"
+#pragma once
 
-#include <iosfwd>
-#include <memory>
-#include <string>
-#include <vector>
```

Comment:
Right now a lot of these includes will be implicitly included via the U_Pw_base_element.hpp. I would like to actually keep the ones that are still used in this class, like e.g. "includes/serializer.h". That way, if the U_Pw_base_element.hpp, or the Element.h class decides it doesn't need a serializer anymore, it won't break this class. Probably the serializer is not a great example, but keeping the includes we actually use will avoid breaking the compilation when some upstream header decides a certain include is no longer needed.

Reply:
Thank you very much for the explanation, reverted.